petergreve/application/test/stores/message-store.js

```diff
@@ -0,0 +1,35 @@
+var rewire = require("rewire");
+var Constants = require('../../app/constants/constants')
+var MessageStore = rewire('../../app/stores/message-store')
+import Dispatcher from '../../app/dispatcher/dispatcher'
+
+describe('Message Store methods', () => {
+	
+	before(() => {
+        MessageStore.__set__('_messages',[])

```

Comment:
Side-note: Whenever `rewire` is called, a new instance of the module is returned. Which can be handy to call `rewire` in the `beforeEach` if you want a full reset instead of emptying all the vars 1 by 1.